.travis.yml

```diff
@@ -213,8 +213,8 @@ script:
       travis_run_in_fold "make.check-qa" make check-qa
     fi
   - |
-    set -e
     if [ "$TEST_PREV_COMMITS" = 1 ] && ! [ "$TRAVIS_PULL_REQUEST" = false ]; then
+      set -e
```

Comment:
The last commit doesn't do what the commit message says and the ones before sound like they should be squashed.

Reply:
Sure.. :)